eurofaang_ri_be/eurofaang_ri_be/settings.py

```diff
@@ -13,6 +13,10 @@
 
 from pathlib import Path
 
+from dotenv import load_dotenv
+
+load_dotenv()
+
```

Comment:
no need to load env vars from inside .env file, these variables are already defined as secrets in Cloud Run